webpack.dev.js

```diff
@@ -22,6 +22,12 @@ export default merge(common, {
     devServer: {
         static: {
             directory: path.join(__dirname, './'),
+            watch: {
+                ignored: [
+                    // ignore changes in '.git' subdirectory (prevent constant hot-reloading in auto-fetch configurations)
+                    '**/.git',
```

Comment:
```suggestion
                    '**/.git',
                    /node_modules/,
		    /coverage/,

```
I noticed that webpack triggers a rebuild when doing unit tests too, adding these two ignore seems to fix it.